crates/forge/src/collecting.rs

```diff
@@ -8,16 +8,17 @@ use rayon::prelude::{IntoParallelRefIterator, ParallelIterator};
 use test_collector::{collect_tests, LinkedLibrary, TestCase};
 use walkdir::WalkDir;
 
+#[cfg_attr(test, derive(Clone))]
```

Comment:
To be able to use clone, but only in tests

Reply:
I think we could derive it for all cases, doesn't matter that much.

Reply:
I would leave it as it is

Reply:
This just adds complication to the code without any benefit.